### PR TITLE
ign -> gz Macro Migration : gz-transport

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/gazebo-tooling/release-tools/issues/737

### CODE REVIEW

File: include/ignition/transport/Helpers.hh [hunk 1]
```diff
@@ -17,3 +17,21 @@
 
 #include <gz/transport/Helpers.hh>
 #include <ignition/transport/config.hh>
+
+// Avoid using deprecated message send/receive function when possible.
+#if ZMQ_VERSION > ZMQ_MAKE_VERSION(4, 3, 1)
+  #define GZ_ZMQ_POST_4_3_1 GZ_ZMQ_POST_4_3_1
```
Comment (chapulina) [inline include/ignition/transport/Helpers.hh:23]:
```suggestion
  #define IGN_ZMQ_POST_4_3_1 GZ_ZMQ_POST_4_3_1
```

File: src/cmd/ign.hh [hunk 2]
```diff
@@ -92,6 +92,7 @@ extern "C" void cmdTopicEcho(const char *_topic, const double _duration,
 
 /// \brief External hook to read the library version.
 /// \return C-string representing the version. Ex.: 0.1.2
-extern "C" const char *ignitionVersion();
+extern "C" const char *gzVersion();
+extern "C" const char GZ_DEPRECATED(12) *ignitionVersion();
```
Comment (chapulina) [inline src/cmd/ign.hh:96]: No need to tick-tock, this isn't installed

**Review by chapulina — CHANGES_REQUESTED** [review]
Just 2 minor comments

File: src/cmd/ign.hh (hunk 2 above)
Comment (methylDragon) [reply src/cmd/ign.hh]: https://github.com/gazebosim/gz-transport/pull/319/commits/3ea20c5946ee6d25084394f60b2a77a3ad8c52a3 !

File: include/ignition/transport/Helpers.hh (hunk 1 above)
Comment (methylDragon) [reply include/ignition/transport/Helpers.hh]: https://github.com/gazebosim/gz-transport/pull/319/commits/3ea20c5946ee6d25084394f60b2a77a3ad8c52a3 !

**Review by chapulina — APPROVED** [review]

**Comment by methylDragon** [comment]
@osrf-jenkins run tests please!

**Pull request merged**